src/app/api/upload/route.ts

```diff
@@ -0,0 +1,34 @@
+import { NextRequest, NextResponse } from "next/server"
+import path from "path"
+import fs from "fs"
+
+// TODO Change this with docker and use docker storage
+//  so it dies with the container? or is this fine?
+const UPLOAD_DIR = path.resolve(process.env.ROOT_PATH ?? "", "public/uploads")
```

Comment:
I think once we containerize the app it will automatically use the path in the `docker fs`.